### PR TITLE
Prepare repo migration

### PR DESCRIPTION
**Pull request by ameukam**
Prepare repo for migration to Github org kubernetes-sigs

### CODE REVIEW

**Comment by ameukam** [comment]
cc @aojea @mneverov

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2023 The Kubernetes Authors.
```
Comment (mneverov) [inline Makefile:1]: does it make sense to use 2024?

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - ameukam
+  - aojea
+  - mneverov
```
Comment (mneverov) [inline OWNERS:6]: could you please also add sarveshr7 here and below?

**Review by mneverov — APPROVED** [review]
LGTM with a couple of comments

File: Makefile (hunk 1 above)
Comment (ameukam) [reply Makefile:1]: I put the year assuming the file was created in 2023. can out 2024.

File: OWNERS (hunk 2 above)
Comment (ameukam) [reply OWNERS:6]: Done.

**Comment by aojea** [comment]
Once the repo is migrated please announce it in the mailing list so we can celebrate  🥇

**Pull request merged**